README.md

```diff
@@ -18,42 +18,34 @@ Then, select one of the following methods.
 
 ### Using Carthage
 
-Checkout the tag PolyNetSDK-version-5-carthage in the repository.
-
-```shell
-git checkout PolyNetSDK-version-5-carthage
-```
-
-Update the carthage dependencies for the specific Apple platform. For example, iOS for PolyNetSampleSwift.
+Update the carthage dependencies for the specific Apple platform. 
```

Comment:
Navigate to a specific project (PolyNetSampleSwift, PolyNetSampleObjectiveC, etc.). Update the Carthage dependencies for the specific Apple platform.